vignettes/EIRprevmatch.Rmd

```diff
@@ -89,85 +173,209 @@ prev <- lapply(
     )
   }
 )
-
-# create dataframe of initial EIR, output EIR, and prev 2-10 results
-EIR_prev <- cbind.data.frame(init_EIR, output_EIR = unlist(EIR), prev = unlist(prev))
+ 
+# Create dataframe of initial EIR, output EIR, and prev 2-10 results
+malSim_P2E <- cbind.data.frame(init_EIR, EIR = unlist(malSim_EIR), prev = unlist(malSim_prev))
```

Comment:
NICE - print out the whole table (only 7 obs) so readers can visualize the output

Reply:
Done!